waiter/config-full.edn

```diff
@@ -525,6 +525,11 @@
                        ;; flagged as never-passed-health-checks upon failure
                        :failed-check-threshold 5}
 
+ :token-config {
+                ;; Configures the amount of history we store for a token.
+                ;; When it is at least 1, it enables blue/green deployment of services.
```

Comment:
Should we wait until blue/green is enabled to have this in the comments?

Reply:
Removed. Updated comment.